### ISSUE TITLE
Is your example actual?

### ISSUE THREAD

**Issue by EndErr** [sirinibin/yii2rest#2]
I'm wandering if the example is actual.

**Comment by avishopxie** [comment]
???

**Comment by sirinibin** [comment]
Yes. You can check the live demo of the API and it's documentation.thanks

**Comment by sirinibin** [comment]
Check this repo.
https://github.com/sirinibin/Yii2-RESTful-API-with-OAuth2

On 03-Oct-2017 4:46 pm, "sirin k" <sirinibin2006@gmail.com> wrote:

Yes. You can check the live demo of the API and it's documentation.thanks
